### ISSUE TITLE
Is this repository alive?

### ISSUE THREAD

**Issue by Kozzi11** [bestit/flagception-bundle#106]
Hi is this bundle still alive? Or should be forked? We are using this package but it starting to be problem because of its inactivity.
@mKnoop @migo315

**Comment by migo315** [comment]
Hey @Kozzi11, I can't give you any info on that. I am no longer the maintainer due to job changes.

@mKnoop Can you give some info?

**Comment by rawroland** [comment]
Hello. We also use this package and would be sad to see it lose maintenance. I would be ready to jump in with some assistance in tackling tasks and merging open request if necessary.

**Comment by teiling88** [comment]
@migo315 and I are looking for a solution for this project. Maybe we find a solution under another flag ;-)

**Comment by teiling88** [comment]
@migo315 and I are looking for a solution for this project. Maybe we find a solution under another flag ;-)

**Comment by yakobe** [comment]
Agreed, it would be nice to know if it is still being maintained. If not, perhaps the repository could be taken over by someone (or worst case a new fork)?

**Comment by hanishsingla** [comment]
If anyone is stuck with this version, you can check out my fork https://packagist.org/packages/symfgenus/flagception-bundle

Would love to add whatever I can

**Comment by yakobe** [comment]
Thanks @hanishsingla. It would be good if an official handover could be organized so this repository can be marked as abandoned in packaging etc if needed. 
The best solution would be if one of the maintainers here would agree to allowing someone else to take over this repo (or at least manage it). Then nobody needs to change things in their projects.

**Comment by hanishsingla** [comment]
@yakobe, yes, that will be certainly the best solution. However, I doubt if any maintainer even checking this repo anymore

**Comment by yakobe** [comment]
@mKnoop please could you take a quick look here so we can work out how to go forward? If this repository is no longer maintained perhaps it could be handed over? Or marked as abandoned in packagist so another official fork can take over?
